grass7/imagery/i.image.bathymetry/i.image.bathymetry.py

```diff
@@ -290,7 +296,10 @@ def main():
                     '(Rapid_ref.sdf))\n'
             r_file.write(ref_file)
             l = []
-            predict = g.read_command("g.tempfile", pid=os.getpid()).strip() + '.txt'
+            #predict = g.read_command("g.tempfile", pid=os.getpid()).strip() + '.txt'
+            #replace dynamic variable of predict with a static filename until g.tempfile formatting
+            #is cross platform
+            predict="bathytemp.txt"
```

Comment:
could we use tempfile library to get a temporary name? something like os.path.join(tempfile.gettempdir(),  os.getpid()).strip() + '.txt') should work

Reply:
Or something like:
https://grass.osgeo.org/grass76/manuals/libpython/script.html#script.core.tempname
?

Reply:
yes, maybe better

Reply:
Better refer to at least "stable", i.e.

https://grass.osgeo.org/grass78/manuals/libpython/script.html#script.core.tempname